components/director/internal/tenantfetcher/client.go

```diff
@@ -29,9 +29,10 @@ type OAuth2Config struct {
 }
 
 type APIConfig struct {
-	EndpointTenantCreated string `envconfig:"APP_ENDPOINT_TENANT_CREATED"`
-	EndpointTenantDeleted string `envconfig:"APP_ENDPOINT_TENANT_DELETED"`
-	EndpointTenantUpdated string `envconfig:"APP_ENDPOINT_TENANT_UPDATED"`
+	EndpointTenantCreated       string `envconfig:"APP_ENDPOINT_TENANT_CREATED"`
+	EndpointTenantDeleted       string `envconfig:"APP_ENDPOINT_TENANT_DELETED"`
+	EndpointTenantUpdated       string `envconfig:"APP_ENDPOINT_TENANT_UPDATED"`
+	EndpointRuntimeMovedByLabel string `envconfig:"optional,APP_ENDPOINT_RUNTIME_MOVED_BY_LABEL"`
```

Comment:
Because we would not be able to start the program if we don't provide the APP_ENDPOINT_RUNTIME_MOVED_BY_LABEL environment variable if optional is omitted here. In that case, we want to start the program and skip getting events for moved runtimes due to the check that you pointed.